module_intros/environment.EnvironmentCluster.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<p>Should we still attribute the data here? I'm only asking because we have other examples (like PMFTs) that also pull from data, if we want to acknowledge here perhaps we should add it elsewhere as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>